stdlib/public/SDK/Dispatch/Private.swift

```diff
@@ -12,13 +12,13 @@
 
 // Redeclarations of all SwiftPrivate functions with appropriate markup.
 
-@available(*, unavailable, renamed:"DispatchQueue.init(label:attributes:target:)")
+@available(*, unavailable, renamed:"DispatchQueue.init(label:qos:attributes:autoreleaseFrequency:target:)")
 public func dispatch_queue_create(_ label: UnsafePointer<Int8>?, _ attr: __OS_dispatch_queue_attr?) -> DispatchQueue
 {
 	fatalError()
 }
 
-@available(*, unavailable, renamed:"DispatchQueue.init(label:attributes:target:)")
+@available(*, unavailable, renamed:"DispatchQueue.init(label:qos:attributes:autoreleaseFrequency:target:)")
```

Comment:
I don't know if this one will actually produce working Fix-Its, because the # of arguments doesn't match up, but it will still give a reasonable diagnostic.